src/divider.jsx

```diff
@@ -51,7 +42,6 @@ const Divider = ({inset, style, ...other}, {muiTheme = ThemeManager.getMuiTheme(
 
 Divider.propTypes = propTypes;
 Divider.defaultProps = defaultProps;
-Divider.contextTypes = contextTypes;
-Divider.childContextTypes = childContextTypes;
+Divider = muiThemeable(Divider);
 
 export default Divider;
```

Comment:
I tried that originally (and wanted to keep it as const)! But I think our eslinter didn't like that for some reason...let me try again and get the actual error